runtime/tests/runtime_integration_tests.rs

```diff
@@ -94,6 +95,36 @@ js_tests!(ipfs_retrieval_tests);
 test_runner_tests!(passing_tests);
 test_runner_tests!(failing_tests expect_failure);
 
+#[tokio::test]
+async fn typescript_stack_trace_test() -> Result<(), AnyError> {
+    let (_, run_error) = run_js_test_file("typescript_fixtures/typescript_stack_trace.ts").await?;
+    let error = run_error.ok_or_else(|| {
+        anyhow!("The script was expected to throw an error. Success was reported instead.")
+    })?;
+
+    if let Some(CoreError::Js(e)) = any_and_jserrorbox_downcast_ref::<CoreError>(&error) {
+        let actual_error = format_js_error(e);
+        // Strip ANSI codes (colors, styles)
+        let actual_error = console_static_text::ansi::strip_ansi_codes(&actual_error);
+        // Replace current working directory in stack trace file paths with a fixed placeholder
+        let cwd_url = ModuleSpecifier::from_file_path(std::env::current_dir().unwrap()).unwrap();
+        let actual_error = actual_error.replace(cwd_url.as_str(), "file:///project-root");
+        // Normalize line endings to Unix style (LF only)
+        let actual_error = actual_error.replace("\r\n", "\n")
```

Comment:
This is similar to the code we have in `format_test_activities()` below, but not exactly the same. WDYT, is it worth extracting a shared helper function?

https://github.com/CheckerNetwork/zinnia/blob/8680101d4e8a72da317fea905d28e7abd7ef3a3c/runtime/tests/runtime_integration_tests.rs#L153-L189